scripts.js

```diff
@@ -0,0 +1,6 @@
+$(window).on("load",function(){
+    $(".loader-wrapper").fadeOut("slow");
+});
+$(window).on("load",function(){
+    $(".loader-content").fadeOut("slow");
```

Comment:
Wondering why we are including this when there is no `loader-content` class within our HTML

Reply:
Hey @yangc95 I have removed now loader-content. Review this!